lib/utils/helpers.ts

```diff
@@ -84,18 +85,21 @@ export function getManifest(
 	image: string,
 	deviceType: string,
 ): Promise<BalenaSdk.DeviceType> {
-	const imagefs = require('resin-image-fs');
-	// Attempt to read manifest from the first
-	// partition, but fallback to the API if
-	// we encounter any errors along the way.
-	return imagefs
-		.readFile({
-			image,
-			partition: 1,
-			path: '/device-type.json',
-		})
-		.then(JSON.parse)
-		.catch(() => balena.models.device.getManifestBySlug(deviceType));
+	const init: typeof _deviceInit = require('balena-device-init');
```

Comment:
Nit: If you make this function `async`, then this can become:

```typescript
const init = await import('balena-device-init');
```

and that way it'll automatically pick up the types by itself, so you can drop the `_deviceInit` import.

That would also let you use `await` below to flatten the promise chain there, which would make that a little clearer too.

The one constraint is that you can't import `Bluebird` as `Promise` any more in that case. Elsewhere we've just been renaming the import to `Bluebird`.